susemanager/src/mgr_bootstrap_data.py

```diff
@@ -893,5 +997,15 @@
         'BASECHANNEL' : 'ubuntu-18.04-pool-amd64-uyuni', 'PKGLIST' : PKGLISTUBUNTU1804,
         'DEST' : '/srv/www/htdocs/pub/repositories/ubuntu/18/4/bootstrap/',
         'TYPE' : 'deb'
-    }
+    },
+    'debian9-amd64-uyuni' : {
```

Comment:
To be honest, if we are going to have `debian9-amd64-uyuni` here, I wonder if we shouldn't have the `uyuni` prefix at `spacewalk-common-channels` repositories as well, in preparation for a future Debian support at SUSE Manager.

@paususe?

Reply:
Yes

Reply:
just to clarify, shall we leave the suffix `-uyuni` as it is in `mgr_bootstrap_data.py`, is it ok?

Reply:
yes!